### ISSUE TITLE
Transcription of the nz.js(con) videos?

### ISSUE THREAD

**Issue by PrototypeAlex** [JavaScript-NZ/Society-Meetings#58]
Might be something worth considering if there's anything left over in the conference kitty?

Not sure if there's a service, or we could just pay someone to do it? But I'm guessing someone knows how it's done.

**Comment by buildmaster** [comment]
awaiting auto transcribe videos to check quality

**Comment by veb** [comment]
Has anyone thought about captioning the videos? YouTube's auto captioning is pretty awful at times. By captioning the content, you'll be making the content far more accessible to both people with hearing impairments and foreigners who can then easily translate the content if they wish.

**Comment by jenofdoom** [comment]
Hi @veb, yep we're definitely seeing what we can get sorted, hence this ticket :)

As a matter of fact I fixed up the captions for one video (just the intro so far) last night. Will keep you all posted on further progress.

**Comment by veb** [comment]
Awesome news and really good to hear you are already on top of this. I'm more than happy to be a test guinea pig to make sure any captions are up to snuff if necessary! :)

**Comment by jenofdoom** [comment]
My apologies for leaving this fallow so long. I did not have the time to progress further with the transcribing, and it seems to be difficult to open open to others helping.

We discussed this issue at the committee meeting on 19/03/19 and decided that we would make this a higher priority for the next conference - potentially by hiring a real time transcription service during the conference as this could help both with the videos and attendees in person as well.
